tiledb/sm/c_api/tiledb.h

```diff
@@ -900,6 +900,14 @@ TILEDB_EXPORT void tiledb_config_free(tiledb_config_t** config);
  *
  * **Parameters**
  *
+ * - `sm.array.timestamp_start` <br>
+ *    When set, an array will be opened betwen this value and
```

Comment:
- Typo on `betwen`.
- We need to explicitly document that these bounds are inclusive.